Title: IPMI Management board: Fixed "cmk -d" on hosts with enabled mgmt board
Level: 1
Component: core
Class: fix
Compatible: compat
Edition: cre
State: unknown
Version: 1.6.0b1
Date: 1535462867

An exception occurred on hosts having IPMI management board enabled when executing
"cmk -d [hostname]" on the console:

C+:
OMD[stable]:~$ cmk --debug -d ipmi-host
Traceback (most recent call last):
  File "/omd/sites/stable/bin/cmk", line 96, in <module>
    exit_status = modes.call(o, a, opts, args)
  File "/omd/sites/stable/lib/python/cmk_base/modes/__init__.py", line 80, in call
    return mode.handler_function(*handler_args)
  File "/omd/sites/stable/lib/python/cmk_base/modes/check_mk.py", line 366, in mode_dump_agent
    source_state, source_output, source_perfdata = source.get_summary_result()
  File "/omd/sites/stable/lib/python/cmk_base/data_sources/abstract.py", line 392, in get_summary_result
    return self._summary_result()
  File "/omd/sites/stable/lib/python/cmk_base/data_sources/ipmi.py", line 152, in _summary_result
    return 0, "Version: %s" % self._get_ipmi_version(), []
  File "/omd/sites/stable/lib/python/cmk_base/data_sources/ipmi.py", line 156, in _get_ipmi_version
    section = self._host_sections.sections.get("mgmt_ipmi_firmware")
AttributeError: 'NoneType' object has no attribute 'sections'
C-:
